example/tutorial/tensorflow_example.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import mlflow
+from mlflow import tensorflow, tracking
+import numpy as np
+import pandas as pd
+import shutil
+import tempfile
+import tensorflow as tf
+
+
+def main(argv):
+    # Builds, trains and evaluates a tf.estimator. Then, exports it for inference, logs the exported model 
+    # with MLflow, and loads the fitted model back as a PyFunc to make predictions.
+    (x_train, y_train), (x_test, y_test) = tf.keras.datasets.boston_housing.load_data()
+    # There are 13 features we are using for inference.
+    feat_cols = [tf.feature_column.numeric_column(key="features", shape=(x_train.shape[1],))]
+    feat_spec = {"features":tf.placeholder("float", name="features", shape=[None, x_train.shape[1]])}
+    regressor = tf.estimator.DNNRegressor(
+    hidden_units=[50, 20],
+    feature_columns=feat_cols)
+    train_input_fn = tf.estimator.inputs.numpy_input_fn({"features": x_train}, y_train, num_epochs=None, shuffle=True)
+    tracking.start_run()
+    mlflow.log_param("Hidden Units", [50,20])
```

Comment:
Sorry one more comment - instead of hardcoding hidden units & steps could we store the values in variables & pass them to the `log_param` & `regressor.train` calls etc?